test/browser/spec/ol/layer/vector.test.js

```diff
@@ -193,6 +193,49 @@ describe('ol.layer.Vector', function () {
       });
     });
 
+    it('detects zero opacity images', function (done) {
```

Comment:
Split this into two tests: one for zero opacity images, one for feature styles with a layer style of null.